VAMobile/src/screens/BenefitsScreen/BenefitsScreen.tsx

```diff
@@ -34,12 +34,38 @@ function BenefitsScreen({}: BenefitsScreenProps) {
   const { t } = useTranslation(NAMESPACE.COMMON)
   const navigateTo = useRouteNavigation()
   const claimsInDowntime = useDowntime(DowntimeFeatureTypeConstants.claims)
+  const appealsInDowntime = useDowntime(DowntimeFeatureTypeConstants.appeals)
+  const featureInDowntime = claimsInDowntime || appealsInDowntime
   const { data: userAuthorizedServices } = useAuthorizedServices({ enabled: screenContentAllowed('WG_Benefits') })
-  const { data: claimsData, isLoading: loadingClaimsAndAppeals } = useClaimsAndAppeals('ACTIVE', 1, {
+  const {
+    data: claimsAndAppeals,
+    isFetching: loadingClaimsAndAppeals,
+    isError: claimsAndAppealsError,
+  } = useClaimsAndAppeals('ACTIVE', 1, {
     enabled: (userAuthorizedServices?.claims || userAuthorizedServices?.appeals) && !claimsInDowntime,
```

Comment:
Should this be...if they aren't BOTH in downtime?

Reply:
Should potentially include the waygate check too?

Reply:
```suggestion
    enabled: ((userAuthorizedServices?.claims && !claimsInDowntime) || (userAuthorizedServices?.appeals && !appealsInDowntime)),
```

Reply:
Good point, added the waygate check and your suggested logic updates